services/scabbard/libscabbard/src/client/mod.rs

```diff
@@ -68,7 +64,15 @@ impl ScabbardClient {
         let body = batches.into_bytes()?;
 
         debug!("Submitting batches via {}", url);
-        let request = Client::new().post(url).body(body);
+        // Allowing unused_mut because request must be mutable if experimental feature
```

Comment:
These changes to the requests should be in the commit that adds the `auth` field to the client. The cleanest way to break this up would probably be to remove all auth stuff from the builder commit, then add the auth field to the builder, client, and requests in the 2nd commit.

Reply:
Updated commits